runtime/opt/zalando/runtime/Docker

```diff
@@ -29,6 +30,15 @@ def get_env_options(config: dict):
         yield '-e'
         yield 'ETCD_URL=http://172.17.42.1:2379'
 
+    yield '-e'
+    yield 'APPLICATION_ID={}'.format(config.get('application_id'))
+
+    yield '-e'
+    yield 'APPLICATION_VERSION={}'.format(config.get('application_version'))
+
+    yield '-e'
+    yield 'INSTANCE_ID={}'.format(boto.utils.get_instance_metadata()['instance-id'])
```

Comment:
I believe "INSTANCE_ID" should not be passed automatically every time (what's the use case? if it's needed you can use the EC2 metadata service to fetch it)
In other words: if we pass the instance-id, why don't we pass all other EC2 metadata? IMHO the EC2 metadata should better be written to the /meta folder